cmd/collectiondir/serve.go

```diff
@@ -455,6 +456,26 @@ func getLimit(c echo.Context, min, defaultLim, max int) int {
 	return lv
 }
 
+type xrpcError struct {
```

Comment:
There is an existing type `xrpc.XRPCError` in `indigo:xrpc`. It is such a simple type that i'm fine re-declaring it though!